File: KingdomCodeTool/wwwroot/Download/202501151754/Angular/shared/DM_BenhVien.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DM_BenhVien } from 'src/app/shared/DM_BenhVien.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DM_BenhVienService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'BenhVien_Id', 'MaBenhVien', 'TenBenhVien', 'TenBenhVien_En', 'TenBenhVien_Ru', 'DiaChi', 'Hang_Id', 'Loai_Id', 'Tuyen_Id', 'CapQuanLy_Id', 'TinhThanhPho_Id', 'TamNgung', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id', 'SiteCode', 'Mapping_Public_Id'];
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DM_BenhVien";
    }
}

File: KingdomCodeTool/wwwroot/Download/202501151754/Service/Implement/DM_BenhVienService.cs
namespace Service.Implement
{
    public class DM_BenhVienService : BaseService<DM_BenhVien, IDM_BenhVienRepository>
    , IDM_BenhVienService
    {
    private readonly IDM_BenhVienRepository _DM_BenhVienRepository;
    public DM_BenhVienService(IDM_BenhVienRepository DM_BenhVienRepository) : base(DM_BenhVienRepository)
    {
    _DM_BenhVienRepository = DM_BenhVienRepository;
    }
    }
    }

